utils/lists.ts

```diff
@@ -227,6 +227,11 @@ export const derivativesList: Record<string, string>[] = [
     description: "Adventurers with distinct but flexible aspects for inspiring stories",
     url: "https://opensea.io/collection/adventurersproject",
   },
+  {
+    name: "⚔️ Battle! (for Loot)",
```

Comment:
Move it below project Banquet to follow alphabetic ordering?

Reply:
Indeed 👀
Thanks